app/src/main/java/io/left/meshim/activities/MainActivity.java

```diff
@@ -176,6 +185,14 @@ public void updateInterface() {
                 mOnlineUserListAdapter.notifyDataSetChanged();
                 mConversationListAdapter.updateList(mService);
                 mConversationListAdapter.notifyDataSetChanged();
+                //notify user of new messages.
+                TextView newMessageNotification =
```

Comment:
Checkstyle has equals sign on second line.

Reply:
weird it gives me a checkstyle error when i put = on second line.

Reply:
:thinking:

Ah well, doesn't really matter. We'll catch it if it does when we do a checkstyle cleanup.